fbw-a32nx/src/systems/fmgc/src/efis/EfisCommon.ts

```diff
@@ -25,6 +25,9 @@ export function withinEditArea(lla: Coordinates, range: EfisNdRangeValue, mode:
 export function calculateEditArea(range: EfisNdRangeValue, mode: EfisNdMode): [number, number, number] {
     switch (mode) {
     case EfisNdMode.ARC:
+        if (range <= 0.5) {
```

Comment:
The FMS doesn't need to know about OANS ranges as it doesn't transmit symbols while OANS is active as far as I know. Or at least they aren't displayed.

Reply:
Also I think rather than editing the edit areas (ha!) each aircraft should have their own. The current definitions are exact for the A320 family FMS and with this PR they would no longer be.